packages/web/src/pages/audio-rewards-page/components/WalletManagementTile.tsx

```diff
@@ -348,38 +350,44 @@ export const WalletManagementTile = () => {
         </div>
       </div>
       <div className={styles.container}>
-        {!isMobileWeb() && onRampProviders[primaryProvider].isEnabled ? (
+        {!isManagedAccount &&
+        !isMobileWeb() &&
+        onRampProviders[primaryProvider].isEnabled ? (
           <OnRampTooltipButton
             provider={primaryProvider}
             isDisabled={!isAnyProviderAllowed}
             bannedState={onRampProviders[primaryProvider].bannedState}
           />
         ) : null}
-        <WalletActions />
+        {!isManagedAccount ? (
```

Comment:
Will note that we did say in the spec doc that you should be able to "receive audio", i.e. we should have a way to see the wallet address.

Reply:
we said during our meeting to remove the buttons. 🤷 you can always go to their page and just tip them from your own acct